pkg/network/network_static.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024 Nubificus LTD.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package network
+
+import (
+	"fmt"
+	"os/user"
+	"strconv"
+	"strings"
+
+	"github.com/nubificus/urunc/internal/constants"
+	"github.com/vishvananda/netlink"
+)
+
+var StaticIPAddr = fmt.Sprintf("%s/24", constants.StaticNetworkTapIP)
+
+type StaticNetwork struct {
+}
+
+func (n StaticNetwork) NetworkSetup() (*UnikernelNetworkInfo, error) {
+	err := ensureEth0Exists()
+	if err != nil {
+		return nil, err
+	}
+	redirectLink, err := netlink.LinkByName(DefaultInterface)
+	if err != nil {
+		return nil, err
```

Comment:
Maybe consider adding a log message?

Reply:
Yes, it could prove useful while debugging. I added a log message.